teamsyncd/teamsync.cpp

```diff
@@ -212,6 +212,8 @@ void TeamSync::removeLag(const string &lagName)
 
 void TeamSync::cleanTeamSync()
 {
+    SWSS_LOG_NOTICE("Cleaning up kernel LAGs ...");
```

Comment:
Ideally removeLag() is getting triggered when the kernel LAG interface is deleted. In cleanTeamSync() we are actually cleaning up some teamd resources by calling removeLag() explicitly for all LAG interfaces which are still present.

The Kernel LAG interfaces are removed actually when we do a kill of teamd process in teammgrd process in cleanTeamProcesses()

So the NOTICE message may not be the right wording. If we need to keep it, we could put it as "Cleaning up LAG teamd resources" or so.

Reply:
Changed as suggested.